recipes-tag/bluesky-browser/meta.yaml

```
@@ -0,0 +1,41 @@
+{% set version = "0.0.1" %}
+{% set setup_py = load_setup_py_data() %}
+
+package:
+    name: 'bluesky-browser'
+    version: {{ version }}
+
+source:
+    git_url: https://github.com/bluesky/bluesky-browser
+    git_rev: v{{ version }}
+
+build:
+    number: 0
+    script: pip install -e .
```

I'd recommend to use [conda-forge's approach](http://conda-forge.org/docs/maintainer/adding_pkgs.html#use-pip), i.e.:
```suggestion
    script: "{{ PYTHON }} -m pip install . --no-deps -vv"
```